packages/utils/src/verify-message.ts

```diff
@@ -0,0 +1,68 @@
+import {
+    SignedMessage,
```

Comment:
I need this to be a shared type, could put it in `types`

`SignedMessage` + `Nep413Message` or just `Message`

Reply:
+1 for putting shared types on the `types` package